app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/destination/ChangeAddressOfferDestination.kt

```diff
@@ -51,64 +48,72 @@ import androidx.compose.ui.tooling.preview.Preview
-import com.hedvig.android.core.designsystem.component.button.LargeTextButton
+import coil.ImageLoader
+import com.hedvig.android.core.designsystem.component.button.HedvigContainedButton
+import com.hedvig.android.core.designsystem.component.button.HedvigTextButton
 import com.hedvig.android.core.designsystem.component.card.HedvigCard
 import com.hedvig.android.core.designsystem.component.card.HedvigInfoCard
 import com.hedvig.android.core.designsystem.material3.squircleMedium
 import com.hedvig.android.core.designsystem.theme.HedvigTheme
 import com.hedvig.android.core.ui.ValidatedInput
-import com.hedvig.android.core.ui.card.ExpandablePlusCard
+import com.hedvig.android.core.ui.appbar.m3.TopAppBarActionType
 import com.hedvig.android.core.ui.dialog.ErrorDialog
+import com.hedvig.android.core.ui.infocard.VectorInfoCard
+import com.hedvig.android.core.ui.preview.PreviewImageLoader
 import com.hedvig.android.core.ui.scaffold.HedvigScaffold
 import com.hedvig.android.core.ui.text.HorizontalItemsWithMaximumSpaceTaken
 import com.hedvig.android.core.uidata.UiMoney
 import com.hedvig.android.feature.changeaddress.ChangeAddressUiState
 import com.hedvig.android.feature.changeaddress.ChangeAddressViewModel
 import com.hedvig.android.feature.changeaddress.data.MoveIntentId
 import com.hedvig.android.feature.changeaddress.data.MoveQuote
-import com.hedvig.android.feature.changeaddress.ui.QuoteCard
-import com.hedvig.android.feature.changeaddress.ui.offer.Faqs
+import com.hedvig.android.feature.changeaddress.data.documentDisplayName
+import com.hedvig.android.feature.changeaddress.ui.offer.QuoteCard
 import hedvig.resources.R
 import kotlinx.coroutines.launch
 import kotlinx.datetime.Clock
 import kotlinx.datetime.TimeZone
 import kotlinx.datetime.toLocalDateTime
-import octopus.type.CurrencyCode
 
 @Composable
 internal fun ChangeAddressOfferDestination(
   viewModel: ChangeAddressViewModel,
   openChat: () -> Unit,
-  navigateBack: () -> Unit,
-  onChangeAddressResult: () -> Unit,
+  close: () -> Unit,
+  onChangeAddressResult: (String?) -> Unit,
+  openUrl: (String) -> Unit,
+  imageLoader: ImageLoader,
 ) {
   val uiState: ChangeAddressUiState by viewModel.uiState.collectAsStateWithLifecycle()
   val moveResult = uiState.successfulMoveResult
 
   LaunchedEffect(moveResult) {
     if (moveResult != null) {
-      onChangeAddressResult()
+      onChangeAddressResult(uiState.movingDate.input?.toString())
     }
   }
   ChangeAddressOfferScreen(
     uiState = uiState,
     openChat = openChat,
-    navigateBack = navigateBack,
+    close = close,
     onErrorDialogDismissed = viewModel::onErrorDialogDismissed,
     onExpandQuote = viewModel::onExpandQuote,
     onConfirmMove = viewModel::onConfirmMove,
+    openUrl = openUrl,
+    imageLoader = imageLoader,
   )
 }
 
 @Composable
 private fun ChangeAddressOfferScreen(
   uiState: ChangeAddressUiState,
   openChat: () -> Unit,
-  navigateBack: () -> Unit,
+  close: () -> Unit,
```

Comment:
Regarding the `close` name change in some places.
Usually `navigateUp` is used explicitly in the top-left icon in screens, so that we can call `navController.navigateUp()`, which basically pops the backstack, but ensures we do not leave the app by doing so.
The name `close` kinda loses this meaning, is it that these screens are not possible to go back from? Is going back supposed to be a non allowed step in this process of moving? Could we in some way preserve being able to go back in those screens?

If there's a good reason why back doesn't work (and in this case I assume pressing the normal back android button/gesture also needs to do a custom thing) then we could rename this to something a bit more descriptive than close?

But since I don't see any `BackHandler` calls here in the first place, maybe we don't need to break back handling from the top left button? Please fill me in here.